pennylane/pauli/pauli_arithmetic.py

```diff
@@ -227,14 +227,23 @@ def operation(self, wire_order=None):
         factors = [op_map[op](wire) for wire, op in self.items()]
         return factors[0] if len(factors) == 1 else prod(*factors)
```

Comment:
In my mind, `Sum` is to `Hamiltonian` as `Prod` is to `Tensor`. So, I would instead expect that keyword toggle to go on the `operation()` method. Is there a reason you put in on `hamiltonian()`? I think it especially makes more sense because then this suggested change is the only thing needed (plus the keyword arg being moved over).
```suggestion
        merge = Tensor if get_as_tensor else prod
        return factors[0] if len(factors) == 1 else merge(*factors)
```
damn git screwed that diff up badly...

Reply:
if not this, I still don't think I'd change the `hamiltonian` method to return a not-Hamiltonian ever. I'd make a whole new method (like.. `tensor()`!)

Reply:
Good call, I tunnel visioned on the hamiltonian method